project/MimaFilters.scala

```diff
@@ -28,65 +28,9 @@ object MimaFilters {
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropOldAndSignal.this"),
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropOldAndSignal.copy"),
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropOldAndSignal.copy$default$2"),
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropOldAndSignal.onOverflow"),
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropNewAndSignal.this"),
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropNewAndSignal.copy"),
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropNewAndSignal.copy$default$2"),
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropNewAndSignal.onOverflow"),
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropNewAndSignal.apply"),
-    exclude[IncompatibleSignatureProblem]("monix.reactive.OverflowStrategy#DropNewAndSignal.unapply"),
-    exclude[IncompatibleMethTypeProblem]("monix.reactive.ObservableLike.apply"),
-    exclude[IncompatibleMethTypeProblem]("monix.eval.TaskLike.apply"),
-    exclude[DirectMissingMethodProblem]("monix.eval.TaskLike.fromComonad"),
-    exclude[IncompatibleMethTypeProblem]("monix.eval.CoevalLike.apply"),
-    exclude[DirectMissingMethodProblem]("monix.eval.CoevalLike.fromComonad"),
-    exclude[IncompatibleMethTypeProblem]("monix.eval.TaskLift.apply"),
-    exclude[IncompatibleMethTypeProblem]("monix.execution.exceptions.ExecutionRejectedException.apply"),
-    exclude[IncompatibleMethTypeProblem]("monix.execution.exceptions.DummyException.apply"),
-    exclude[IncompatibleMethTypeProblem]("monix.execution.exceptions.APIContractViolationException.apply"),
-    exclude[IncompatibleMethTypeProblem]("monix.execution.exceptions.BufferOverflowException.apply"),
-    exclude[IncompatibleMethTypeProblem]("monix.execution.exceptions.DownstreamTimeoutException.apply"),
-    exclude[IncompatibleMethTypeProblem]("monix.execution.exceptions.CompositeException.apply"),
-    exclude[IncompatibleMethTypeProblem]("monix.execution.exceptions.UpstreamTimeoutException.apply"),
-    exclude[IncompatibleMethTypeProblem]("monix.execution.schedulers.StartAsyncBatchRunnable.apply"),
-    exclude[IncompatibleSignatureProblem]("monix.execution.schedulers.ReferenceScheduler#WrappedScheduler.unapply"),
-    exclude[IncompatibleResultTypeProblem]("monix.execution.Scheduler.Implicits"),
-    exclude[IncompatibleResultTypeProblem]("monix.tail.batches.EmptyBatch.cursor"),
-    exclude[IncompatibleResultTypeProblem]("monix.tail.batches.EmptyCursor.next"),
-    exclude[IncompatibleMethTypeProblem]("monix.catnap.FutureLift.apply"),
+    // Breaking changes for https://github.com/monix/monix/pull/946
+    exclude[IncompatibleResultTypeProblem]("monix.execution.misc.Local.defaultContext"),
```

Comment:
#946 : Changed `def defaultContext()` inferring to `Context.Unbound` to `Local.Context`